src/overlays/actors/ovl_En_Hint_Skb/z_en_hint_skb.c

```diff
@@ -143,19 +143,19 @@ typedef enum EnHinkSkbAnimation {
 } EnHinkSkbAnimation;
 
 static AnimationInfo sAnimationInfo[ENHINTSKB_ANIM_MAX] = {
-    { &object_skb_Anim_0064E0, 0.96f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },      // ENHINTSKB_ANIM_0
-    { &object_skb_Anim_003584, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },       // ENHINTSKB_ANIM_1
-    { &object_skb_Anim_002190, 0.6f, 0.0f, 0.0f, ANIMMODE_ONCE_INTERP, 4.0f }, // ENHINTSKB_ANIM_2
-    { &object_skb_Anim_002AC8, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },       // ENHINTSKB_ANIM_3
-    { &object_skb_Anim_00270C, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -4.0f },       // ENHINTSKB_ANIM_4
-    { &object_skb_Anim_00697C, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // ENHINTSKB_ANIM_5
-    { &object_skb_Anim_006D90, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // ENHINTSKB_ANIM_6
-    { &object_skb_Anim_001D1C, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },       // ENHINTSKB_ANIM_7
-    { &object_skb_Anim_003584, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // ENHINTSKB_ANIM_8
-    { &object_skb_Anim_003584, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -16.0f },      // ENHINTSKB_ANIM_9
-    { &object_skb_Anim_002AC8, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // ENHINTSKB_ANIM_10
-    { &object_skb_Anim_0015EC, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -8.0f },       // ENHINTSKB_ANIM_11
-    { &object_skb_Anim_0009E4, 1.0f, 0.0f, 0.0f, ANIMMODE_LOOP, -8.0f },       // ENHINTSKB_ANIM_12
+    { &gStalchildWalkAnim, 0.96f, 0.0f, 0.0f, ANIMMODE_LOOP, -4.0f },        // STALCHILD_ANIM_WALK
+    { &gStalchildStandUpAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },      // STALCHILD_ANIM_STAND_UP_MORPH_1
```

Comment:
Since you change the enum names here (`STALCHILD_ANIM_`), could you change the enum itself instead of desyncing these commented enums?

Reply:
Ah, sorry I didn't understand this file very well, now I get it -- it's being generated from the Enum names above.

I changed the enum to include those names.